package.json

```diff
@@ -19,13 +19,28 @@
   "license": "ISC",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/blackbaud/skyux-sample.git"
+    "url": "git+https://github.com/blackbaud/skyux-list-builder-view-grids.git"
   },
   "bugs": {
-    "url": "https://github.com/blackbaud/skyux-sample/issues"
+    "url": "https://github.com/blackbaud/skyux-list-builder-view-grids/issues"
+  },
+  "homepage": "https://github.com/blackbaud/skyux-list-builder-view-grids#readme",
+  "peerDependencies": {
+    "@angular/common": "^4.3.6",
+    "@angular/core": "^4.3.6",
+    "@skyux/animations": ">=3.0.0-rc.0",
+    "@skyux/forms": ">=3.0.0-rc.0",
+    "@skyux/grids": "3.0.0-alpha.1",
```

Comment:
Some of these peer dependencies need carets `^`.

Reply:
got em.